packages/opentelemetry-host-metrics/README.md

```diff
@@ -39,6 +39,39 @@ const hostMetrics = new HostMetrics({ meterProvider, name: 'example-host-metrics
 hostMetrics.start();
 ```
 
+## Semantic Conventions
+
+This package uses Semantic Conventions [Version 1.25.0](https://github.com/open-telemetry/semantic-conventions/tree/v1.25.0/docs/system).
+As for now the Semantic Conventions are bundled in this package but eventually will be imported from `@opentelemetry/semantic-conventions` package when it is updated to latest version.
+Ref: [opentelemetry-js/issues/4235](https://github.com/open-telemetry/opentelemetry-js/issues/4235)
+
+Metrics collected:
+
+| Metric                      | Short Description                                         |
+| --------------------------- | --------------------------------------------------------- |
+| `system.cpu.time`           | Seconds each logical CPU spent on each mode               |
+| `system.cpu.utilization`    | Difference in system.cpu.time since the last measurement  |
+| `system.memory.usage`       | Reports memory in use by state                            |
+| `system.memory.utilization` |                                                           |
+| `system.network.dropped`    | Count of packets that are dropped                         |
+| `system.network.errors`     | Count of network errors detected                          |
+| `system.network.io`         |                                                           |
+| `process.cpu.time`          | Total CPU seconds                                         |
+| `process.cpu.utilization`   | Difference in process.cpu.time since the last measurement |
+| `process.memory.usage`      | The amount of physical memory in use                      |
+
+Attributes collected:
+
+| Metric                      | Short Description                  | Notes |
```

Comment:
Since there is nothing on the Notes column, it can be removed

Reply:
Applied